src/pages/Singlepokemon.js

```diff
@@ -19,18 +19,50 @@ export default function Singlepokemon() {
   }, [id]);
 
   function Renderpokemon() {
+    let history = useHistory();
+    function goback() {
+      return history.push("/pokemon");
+    }
+    function loadprevious() {
+      return history.push(`/pokemon/${id - 1}`);
```

Comment:
would the loadprevious work if the user receives the link of the 25th pokemon - i think no, id is not a number but a string. So instead of returning the 24th pokemon, it would return 24-1... i think your version only works if the user starts with #1. because its ithen converted to a number, when he goes to the #2 pokemon.